index.html

```diff
@@ -48,37 +48,33 @@
 
   // editors, add as many as you like
   // only "name" is required
-  editors: [      {
-      name: "Manu Sporny",
-      url: "http://manu.sporny.org/",
-      company: "Digital Bazaar",
-      companyURL: "https://digitalbazaar.com/",
-      w3cid: 41758
-    },
-    {
-      name: "Markus Sabadello",
-      url: "https://www.linkedin.com/in/markus-sabadello-353a0821",
-      company: "Danube Tech",
-      companyURL: "https://danubetech.com/",
-      w3cid: 46729
-    }, {
-      name: "Drummond Reed",
-      url: "https://www.linkedin.com/in/drummondreed/",
-      company: "Evernym",
-      companyURL: "https://www.evernym.com/",
-      w3cid: 3096
-    }
-  ],
+  editors: [{
+    name: "Orie Steele",
+    url: "https://www.linkedin.com/in/or13b/",
+    company: "Transmute",
+    companyURL: "https://www.transmute.industries/"
+  }, {
+    name: "Manu Sporny",
+    url: "http://manu.sporny.org/",
+    company: "Digital Bazaar",
+    companyURL: "https://digitalbazaar.com/",
+    w3cid: 41758
+  }],
 
   // authors, add as many as you like.
   // This is optional, uncomment if you have authors as well as editors.
   // only "name" is required. Same format as editors.
   authors: [{
-      name: "Manu Sporny",
-      url: "http://manu.sporny.org/",
-      company: "Digital Bazaar",
-      companyURL: "https://digitalbazaar.com/",
-      w3cid: 41758
+    name: "Orie Steele",
+    url: "https://www.linkedin.com/in/or13b/",
+    company: "Transmute",
+    companyURL: "https://www.transmute.industries/"
```

Comment:
@OR13, make sure to add your `w3cid` here as well.

Reply:
For info, the w3cid value is 109171